### DISCUSSION TITLE
How can I creat a new material using PyAEDT?

### DISCUSSION THREAD

**Discussion by Dicksky** [ansys/pyaedt#3920] [Q&A] [answered]
Hello PyAEDT team, I would like to create a material with a certain loss tangent value, how should I do that? Thanks a lot:)

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @Dicksky,

Please try:

```
import pyaedt

version = '2023.2'

app = pyaedt.Hfss(specified_version=version)

mat = app.materials.add_material(materialname="amazing_material")
mat.dielectric_loss_tangent = 0.2


app.release_desktop(False, False)
```

For more information:
https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.modules.Material.Material.html
